graph/schema.graphqls

```diff
@@ -1,69 +1,188 @@
-# in the schema we will be doing Query and Mutations
+"""
+Search Query API.
+"""
 schema { 
   query: Query
   mutation: Mutation
 }
 
-# These are the two queries we will be doing
+"""
+Queries supported by the Search Query API.
+"""
 type Query {
+  """
+  Search for resources index from the managed clusters.  
+  *[PLACEHOLDER] Results only include kubernetes resources for which the authenticated user has list permission.*
+
+  For more information see the feature spec.
+  """
   search(input: [SearchInput]): [SearchResult]
-  messages : [Message]
+  
+  """
+  Query all values for the given property.  
+  If a query is included, the results are filtered to only resources matching the query.  
+  For example, if the property is `name` and the query is `namespace:foo`, the result is the names of all objects in the namespace foo.
+  
```

Comment:
this is a little unclear, isn't the property` name` a part of the query?

Reply:
@Anxhela21 Is this any better?

```
  Query all values for the given property.  
  Optionally, a query can be included to filter the results.  
  For example, if we want to get the names of all resources in the namespace foo, we can pass a query with the filter `{property: namespace, values:['foo']}`
  
  **Default limit is** 10,000  
  A value of -1 will remove the limit. Use carefully because it may impact the service.
```

Reply:
yeah, that's better. it looked like name was apart of the actual query in original.